### PR TITLE
Fabo/fix delegation update issue

### PR DESCRIPTION
**Pull request by faboweb**
Description:

delegations request would update state of next user after sign out sign in

❤️ Thank you!

### CODE REVIEW

File: app/src/renderer/vuex/modules/delegation.js [hunk 1]
```diff
@@ -58,6 +58,7 @@ export default ({ node }) => {
     },
     resetSessionData({ rootState }) {
       rootState.delegation = JSON.parse(JSON.stringify(emptyState))
+      console.log("rest", rootState.delegation)
```
Comment (NodeGuy) [inline app/src/renderer/vuex/modules/delegation.js]: Was this intended for temporary debugging?

File: test/unit/specs/store/delegation.spec.js [hunk 2]
```diff
@@ -3,11 +3,12 @@ import setup from "../../helpers/vuex-setup"
 let instance = setup()
 
 describe("Module: Delegations", () => {
-  let store
+  let store, node
```
Comment (NodeGuy) [inline test/unit/specs/store/delegation.spec.js]: It's better form to use separate lines for each variable:

```JavaScript
let store
let node
```

File: app/src/renderer/vuex/modules/delegation.js (hunk 1 above)
Comment (faboweb) [reply app/src/renderer/vuex/modules/delegation.js]: Removed it

File: test/unit/specs/store/delegation.spec.js (hunk 2 above)
Comment (faboweb) [reply test/unit/specs/store/delegation.spec.js]: To be consistent, I would need to change this everywhere. Let's not do it until we come up for a rule about it, ok?

**Pull request merged**